lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -48,6 +48,8 @@ def self.create_release_upload(api_token, owner_name, app_name)
         when 404
           UI.error("Not found, invalid owner or application name")
           false
+        when 500..
```

Comment:
The endless range was only introduced in Ruby 2.6 (see https://rubyreferences.github.io/rubychanges/2.6.html#endless-range-1 ).
Do we want to force Ruby >=2.6 by using this? 🤔

Reply:
Good find - did not intend that, switched it to a range with an end value.